roles/openshift_persistent_volumes/templates/persistent-volume-claim.yml.j2

```diff
@@ -7,7 +7,7 @@ items:
   kind: "PersistentVolumeClaim"
   metadata:
     name: "{{ claim.name }}"
-{% if claim.annotations %}
+{% if claim.annotations is defined and claim.annotations | length %}
```

Comment:
Not sure if `and claim.annotations | length` is required here - I'd expect it to add an empty `annotation:` section, which is valid.

If you can reliably reproduce it could you try that?

Reply:
I confirm, it works fine with an emtpy `annotation` section. I'm pushing it.